src/views/MainView/MainView.test.ts

```diff
@@ -0,0 +1 @@
+export {}
```

Comment:
Same for this , if we are committing code into the codebase it should probably be tested in the same PR